spec/dummy_app/app/active_record/shrine_versioning_uploader.rb

```diff
@@ -2,17 +2,17 @@ class ShrineVersioningUploader < Shrine
   plugin :activerecord
 
   plugin :cached_attachment_data
-  plugin :delete_raw
   plugin :determine_mime_type
   plugin :pretty_location
-  plugin :processing
   plugin :remove_attachment
-  plugin :versions
 
-  process(:store) do |io|
-    {
-      original: io,
-      thumb: FakeIO.another_version(io, :thumb),
-    }
+  if Gem.loaded_specs['shrine'].version >= Gem::Version.create('3')
```

Comment:
shouldn't there be an `else` with the versions plugin?

Reply:
`thumb` method will work only with derivatives
https://github.com/sferik/rails_admin/blob/e4d970e96a845f20ed898ebd6456062bb13a422c/lib/rails_admin/config/fields/types/shrine.rb#L10-L13

It won't work with version of shrine older than v3. So I just skipped tests about image versions:
https://github.com/sferik/rails_admin/blob/e4d970e96a845f20ed898ebd6456062bb13a422c/spec/rails_admin/config/fields/types/shrine_spec.rb#L14-L20

@prem-prakash, do you think we need to support thumbs for older versions of shrine too?

Reply:
@akhsunna I would say that it would be a good practice at least to not break with older versions, what do you think?

Reply:
Okay, maybe it's worth it :) Will add it soon